drivers/sensor/bmi160/bmi160_trigger.c

```diff
@@ -56,7 +81,10 @@ static void bmi160_handle_interrupts(void *arg)
 	union {
 		u8_t raw[6];
 		struct {
+/* dummy byte in beginning!! not working for I2C, kept to not break exising */
```

Comment:
This problem should be described in more detail, referenced if possible.

Reply:
I don't understand the background of dummy byte. my guess that used for struct padding, if so it should not be at beginning of structure. I don't have bmi160 device to confirm so.